apps/site/src/views/Home/sections/FAQ/FAQ.module.scss

```diff
@@ -0,0 +1,75 @@
+@use "../../lib/styles/zothacks-theme" as theme;
```

Comment:
Nitpick: No need to provide the import path here, since we specified in `next.config.js` to look for Sass imports in the `src/lib/styles/` directory.  Just `"zothacks-theme"` is fine.